src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -121,11 +121,6 @@ struct Params {
     }
 }
 
-enum BodyParamStrategy: String {
```

Comment:
I think we should keep this since and just add unixTime as a case. While it is only used in the strategy property now, it could potentially be used elsewhere in the future and I wouldn't want to have magic strings all over the place (except in the stencil files where's there's no alternative)

Reply:
Good Point. I remove that only because nowhere in the code was using this enum. I add the enum back and change the code to use it.